mikrotik-nat-sync/src/MikroTikNATSyncPlugin.php

```
@@ -0,0 +1,104 @@
+
+    public function getId(): string
+    {
+        return 'mikrotik-nat-sync';
+    }
+
+    public function register(Panel $panel): void
+    {
+        //
+    }
+
+    public function boot(Panel $panel): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$panel'. (undefined)

(UnusedFormalParameter)